src.py

```diff
@@ -0,0 +1,23 @@
+def encrypt(text, shift):
+    result = ""
+    for char in text:
+        if char.isupper():  # Uppercase letters
+            result += chr((ord(char) - 65 + shift) % 26 + 65)
+        elif char.islower():  # Lowercase letters
+            result += chr((ord(char) - 97 + shift) % 26 + 97)
+        else:
+            result += char  # Leave non-alphabetic characters unchanged
+    return result
+
+def decrypt(text, shift):
+    return encrypt(text, -shift)  # Use the same function for decryption
```

Comment:
How this function works should be more documented:
Why the `for` loop?
It should also describe why the operations work, and why the decryption works like it does.